### DISCUSSION TITLE
Webhooks var list

### DISCUSSION THREAD

**Discussion by Mozitux** [AzuraCast/AzuraCast#5431] [Q&A] [answered]
Hello,
I'm trying to use the webhooks' system with Discord, work fine, but where can I find the list of {{ var.name }} to add more information?
Thanks!

**Comment by BusterNeece** [comment, marked as answer]
The variables match the Now Playing API response: https://demo.azuracast.com/api/nowplaying_static/azuratest_radio.json

So, if the path to the JSON variable is `now_playing -> song -> title`, then the variable is `{{ now_playing.song.title }}`.

**Reply by Mozitux** [reply]
Thanks!

**Comment by guiltlab** [comment]
Not what you asked, but I thought it could be useful to note in this topic: if you want to target a discord _thread_ and not a _channel_, you can do it by adding `?thread_id=xxxxxxxx` at the end of your webhook url, replacing xxxxxxxx by the actual thread ID (found by right clicking on the thread > copy ID).

**Comment by R6gear** [comment]
Hi, can you give me some help i want to know wich DJ it's connected and disconnected. i put {{ live.streamer.connected }} but it's not working.

**Comment by kikou2022** [comment]
Hello! 

Is it possible to format (bold, italic, etc.) content sent by email using the integrated Webhook and in this case in what way?

Here is my text in which I want to highlight the name of the station, the title, the artist etc ... Thanks to people :-)

```
ACTUELLEMENT SUR  {{ station.name }} : {{ now_playing.song.title }} DE  {{ now_playing.song.artist }}.

CE TITRE EST SORTI EN {{ now_playing.song.custom_fields.annee }} !

À SUIVRE :  {{ playing_next.song.artist }} !

BONNE JOURNÉE SUR {{ station.url }} ! 
```
